documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -221,18 +221,21 @@ The final version of the evaluation report may only be submitted to the Applican
 8. The Applicant may only make the evaluation report available to third parties in full text, stating the date of issue, and pass it on after written approval by the Evaluation Facility.
 
 ### 3.6 Change Certification     
-1. Change Certifications can be carried out at the Applicant\'s request.
-2. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
-3. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
+1. After a component has been certified, the applicant must inform the Certification Body and the Evaluation Facility (in the case of Assurance Level 2) of any changes made to the component. The applicant must report the changes and fill in the impact assessment provided by the Certification Body.
+2. Change Certifications can also be carried out at the Applicant\'s request.
+3. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
+4. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
```

Comment:
@Sonia-IDSA & @mokamhuber: I don't see any changes in items 4 and 5 related to my comment yet, so I want to elaborate on my point: My understanding was, that if the applicant requests Maintenance, the CB looks at the changes, if necessary asks the EF for input, and decides whether th evaluation can procede as Maintenance or has to be changed to a Re-Evaluation. In case it stays as Maintenance, I see no need to involve the EF any further. If my understanding is correct, i can provide a rephrased text for items 4 and 5. If the current text is current (i.e. the EF is involved during the entire Maintenance), this comment can be closed.

Reply:
The process is different for assurance level 1 and for 2&3. In both cases the trigger can be a change from the applicant´s side (to the component) or a change from IDS side (testbed, certification criteria). If the change comes from the applicant´s side, it is mandatory to inform the CB (AL 1) and the EF (AL1, 2&3). The difference is: in AL1, the applicant conducts impact assessment and sends it to CB. CB decides if re-certification is required. For AL 2&3, the applicant documents changes and sends them to EF an CB. The EF evaluates the changes and sends change evaluation report (specifying the need for maintenance or re-certification) to CB. CB has the final say. So the EF is involved from the beginning in AL2&3. The main reason for this is that the EF has already evaluated the component once, so they are qualified to evaluate the changes.

Reply:
1. You are right about the two scenarios, that trigger a change certification. However, the IDS-related change isn't actually mentione in item 1 and should be added.
2. "it is mandatory to inform the CB (AL 1) and the EF (AL1, 2&3)" I believe this should be AL 1-3 for the CB and AL 2-3 for the EF, which is how it is written in the text, just not your above comment. 
3. I personally do not see the need to involve the EF in case of minor, possibly even non-functional, changes, but at least this way the process is always the same, independent on the type of change. I however don't think, that what you wrote in the comment (from "in AL1, the applicant conducts ..." to "... re-certification) to CB") is currently stated in the text.

Reply:
After further discussion, we decided that we currently plan to include the Evaluation Facility for Change Certification of AL 2 and 3. In case that proves to add too much overhead for maintenance certifications, we can change the Rules of Procedure later on